rskj-core/src/main/java/co/rsk/net/light/LightProcessor.java

```diff
@@ -185,4 +186,30 @@ public void processGetBlockBodyMessage(long id, byte[] blockHash, LightPeer ligh
     public void processBlockBodyMessage(long id, List<BlockHeader> uncles, List<Transaction> transactions, LightPeer lightPeer) {
         throw new UnsupportedOperationException("Not supported BlockBody processing");
     }
+
+    public void processGetStorageMessage(long id, byte[] blockHash, byte[] addressHash, byte[] storageKeyHash, LightPeer lightPeer) {
+        logger.trace("Processing storage request {} block {} address {} storage key {}", id,
```

Comment:
To fix the SonarCloud advice "Invoke method only conditionally", we will start to log these messages in the next manner:

```
String logBlockHash = Hex.toHexString(blockHash);
        logger.trace("Processing block header request {} block {}", id, logBlockHash);
```